posthog/tasks/demo_create_data.py

```diff
@@ -1,7 +1,7 @@
 from celery import shared_task
 from sentry_sdk import capture_exception
 
-from posthog.demo.matrix.manager import MatrixManager
+import posthog.demo.matrix.manager as manager
```

Comment:
```suggestion
from posthog.demo.matrix import manager```

Reply:
Just a heads up, I don't think you actually want those three backticks there. ` ``` `